src/Winium.Desktop.Driver/CommandExecutors/QuitExecutor.cs

```diff
@@ -8,7 +15,26 @@ protected override string DoImpl()
         {
             if (!this.Automator.ActualCapabilities.DebugConnectToRunningApp)
             {
-                if (!this.Automator.Application.Close())
+                // If application had exited, find and terminate all children processes
+                if (this.Automator.Application.HasExited())
```

Comment:
This block code is duplicated 90% with CloseExecutor.cs, if we go for this approach, we should made a helper function and call it in 2 those places to avoid code duplication and easier to maintain.